aexpect/remote.py

```diff
@@ -19,6 +19,27 @@
 
 """
 Functions and classes used for logging into guests and transferring files.
+
+Example usage:
+
+::
+
+    import aexpect
+    import aexpect.remote
+
+    b = aexpect.ShellSession("sh")
+    b.cmd("echo asdf > /tmp/bbb")
+
+    a = aexpect.remote.remote_login("ssh", "127.0.0.1", 5000, "root", "123456", "[\\#\\$]")
+    a.cmd("rm /tmp/aaa -Rf")
+
+    aexpect.remote.scp_to_reusing_session(a, "/tmp/bbb", "/tmp/aaa")
+    aexpect.remote.scp_from_reusing_session(a, "/tmp/aaa", "/tmp/ccc")
+
+    a.close()
+    b.close()
```

Comment:
a usage with `with` would be probably better...

Reply:
Yes, it will also make the usage blocks of each session clearer - I only copy pasted a previous example without taking a deeper look into the example. Added now.